packages/integration/lib/src/bloc/compose/compose_bloc.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:bloc/bloc.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
```

Comment:
we need to import whole core package? can see here we have removed likeminds_feed_fltuter_ui

Reply:
Core package also has an export statement for UI package. We can keep it consistent by importing this one only in all features.

What say?